test/e2e/reconciler/locationworkspace/rootcompute_test.go

```diff
@@ -153,4 +163,80 @@ func TestRootComputeWorkspace(t *testing.T) {
 		}
 		return true, ""
 	}, wait.ForeverTestTimeout, time.Millisecond*100)
+
+	t.Logf("Wait for being able to list Deployments in the user workspace")
+	require.Eventually(t, func() bool {
```

Comment:
We usually try to use `framework.Eventually`. Was there a specific reason not to use it here?

Reply:
I just copied the code above TBF :-)

Reply:
fixed